Modded Ubuntu with pentesting tools features.

Midohajhouj opened the pull request:
I have enhanced the gui.sh script by incorporating essential tools to improve its functionality and user experience. The updates include:
+ Kali Linux Tools Installer: A streamlined way to install essential Kali Linux tools.
+ Ghost Framework: An automated Android Debug Bridge (ADB) control tool for managing Android devices.
+ Improved Error Handling: Enhanced mechanisms to handle errors gracefully, ensuring a smoother user experience.
+ Additional Tools: Added GIMP  (Image editor), htop (System monitor), Python3 (Programming language).

Midohajhouj committed:
Setup (Stable) 

Samsung audio bug fix v2
Bugs fixed
Better error handling

Midohajhouj committed:
Update Uninstaller

Midohajhouj committed:
Update README.md

Midohajhouj committed:
Linux Tools 

Top 40 Kali Linux Pentesting Tools in Your Pocket on this Modded Ubuntu Distro

Midohajhouj committed:
Sudo Add

Midohajhouj committed:
New Features

GIMP (for image editing)
htop (for system monitoring)
Python3 (for programming)
LibreOffice (for office tasks)
Ghost Framework (for security testing)
Firefox Installer on master

Midohajhouj committed:
Update README.md

BDhackers009 commented:
Thanks for your enhancement. Let me review the code and give you the feedbacks if there is any.

BDhackers009 commented:
I have some feedbacks for enhancement please reach out to me in telegram `@bdhackers009`

Midohajhouj commented:
Done

Midohajhouj committed:
New Features

GIMP (for image editing)
htop (for system monitoring)
Python3 (for programming)
Ghost Framework (for security testing)
Firefox Installer on master

Midohajhouj committed:
New Features

Kali Linux Tools (for pentesting)
GIMP (for image editing)
htop (for system monitoring)
Python3 (for programming)
LibreOffice (for office tasks)
Ghost Framework (for security testing)
Firefox Installer on master

Midohajhouj committed:
Sudo Add

Midohajhouj commented:
DONE have a look at #189
